consensus/cp_mainvote.go

```diff
@@ -0,0 +1,60 @@
+package consensus
+
+import (
+	"github.com/pactus-project/pactus/crypto/hash"
+	"github.com/pactus-project/pactus/types/vote"
+)
+
+type cpMainVoteState struct {
+	*consensus
+}
+
+func (s *cpMainVoteState) enter() {
```

Comment:
doesn't need to implement `enter`?

Reply:
Good fine. we can `decide` here. If there is enough vote we can decide on enter.

Reply:
so please implement `enter` and resolve